10up/simple-podcasting
# Fix issues reported for v1.8.1 release

### Description of the Change

This PR fixes some of the issues reported in #313 

The following are the two issues that are fixed in this PR.

1. In the settings for each Podcast you can set a Podcast cover image. Within each Podcast block, you can also set an episode image. The preview within the editor always shows the Podcast cover image, not the Podcast block episode image. It does show properly on the front-end though.
2. I'm getting a PHP warning on the front-end: Attempt to read property "term_id" on string, pointing to includes/blocks/podcast/markup.php:43

Partially Closes #313 

### How to test the Change

Here are the steps to test the first change

- Create a Podcast show term
- Assign an image to the podcast show
- Create a new post
- Add the Podcast show term to the post
- Add a new Podcast block
- Enable the 'Show Display Art' checkbox
- The Podcast block will show the Podcast show image
- Now add a new episode image
- The preview should change and then show the episode image instead of the podcast show image

Here are the steps to test the second change

- Create a new post
- Don't add any Podcast term to the post
- Add a new Podcast block
- Save the post and view it in the frontend
- There should not be any PHP warnings

### Changelog Entry
- Fixed - Show the preview of the episode image if set at block level
- Fixed - PHP warning when an episode is not associated with any Podcast show

### Credits
Props @sudar 

### Checklist:
